example/profile.php

```diff
@@ -16,7 +20,13 @@
 try {
     $yotiClient = new Yoti\YotiClient($config['sdkId'], $config['pemFile']);
     $profile = $yotiClient->getActivityDetails($token);
-    $selfie = base64_encode($profile->getSelfie());
+    $base64Selfie = ActivityDetailsHelper::getBase64Selfie($profile);
+    $selfieEntity = $profile->getSelfieEntity();
+    // Retrieve selfie image format
+    $imageFormat = $selfieEntity ? $selfieEntity->getType() : 'jpeg';
+    $selfieFile = "selfie.{$imageFormat}";
+    // Create selfie image file.
+    file_put_contents($selfieFile, $profile->getSelfie(), LOCK_EX);
```

Comment:
I understand the logic, but I would minimise the code here for simplicity's sake.

```
$base64Selfie = ActivityDetailsHelper::getBase64Selfie($profile);
file_put_contents('selfie.jpeg', $profile->getSelfie(), LOCK_EX);
```

Reply:
I would also use a `uploads` or `public` folder to save the selfie. The content in this folder should be ignored by git, but also add a `.keep` file inside.

Reply:
I wouldn't mind hard coding the image file name, but I was trying to show to the user how they could get the selfie image file format.